### ISSUE TITLE
How to download a pornhub video from an url?

### ISSUE THREAD

**Issue by BruceDai003** [blackmatch/pornhub-downloader#4]
Did you provide a way to download a pornhub video just from one url? Not by searching some keyword and download all the videos from there. I mean download by keyword will probably download a huge amount of videos, I don't want that.

**Comment by blackmatch** [comment]
@BruceDai003 Thank you for feedback!  I pay attention to download more and more videos for now. The idea of you is good. I will think about that.  Please give attention to next release version.

**Comment by BruceDai003** [comment]
Thanks for your quick reply. I didn't use node.js before. So after I typed `npm install` when I directed into the downloaded folder, it seems that the extracting processing got stuck somewhere, and error pops up later.
Here is a screen capture( error will pop up, but not captured in this picture)
![qq 20171114181527](https://user-images.githubusercontent.com/14111577/32774621-d3996676-c967-11e7-92e3-54415895dc1e.png)
Any suggestions?
